openandroidinstaller/assets/configs/kminilte.yaml

```diff
@@ -0,0 +1,28 @@
+metadata:
+  maintainer: Ludovic Rousseau
+  device_name: Samsung Galaxy S5 mini
+  is_ab_device: false
+  device_code: kminilte
+  supported_device_codes:
+    - kminilte
+    - kminiltexx
```

Comment:
It would be nice if you could also add the  `twrp-link` tag, according to https://github.com/openandroidinstaller-dev/openandroidinstaller/blob/main/docs/how_to_contribute_your_own_installation_configurations.md

Reply:
Added